### PR TITLE
Module1 branching task4 (A brick & an aperture)

### PR DESCRIPTION
**Pull request by pmiakish**

### CODE REVIEW

File: module_1/src/com/epam/BranchingTaskD.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.epam;
```
Comment (arturlyup) [inline module_1/src/com/epam/BranchingTaskD.java:1]: move to new package

File: module_1/src/com/epam/BranchingTaskD.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+public class BranchingTaskD {
+    public static void main(String[] args) {
+        Scanner console = new Scanner(System.in).useLocale(Locale.US);
+        System.out.println ("Enter the rectangle aperture's dimensions (A, B): ");
+        System.out.print ("A value: ");
+        double a = console.nextDouble();
+        System.out.print ("B value: ");
+        double b = console.nextDouble();
+        System.out.println ("Enter the brick dimensions (x, y, z): ");
+        double[] brickDimensions = new double[3];
```
Comment (arturlyup) [inline module_1/src/com/epam/BranchingTaskD.java:15]: let's move 3 to some constant like NUMBER_OF_SIDES = 3

File: module_1/src/com/epam/BranchingTaskD.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+        System.out.print ("x value: ");
+        brickDimensions[0] = console.nextDouble();
+        System.out.print ("y value: ");
+        brickDimensions[1] = console.nextDouble();
+        System.out.print ("z value: ");
+        brickDimensions[2] = console.nextDouble();
+        Arrays.sort(brickDimensions);
+        if (Math.min(a,b) >= brickDimensions[0] && Math.max(a,b) >= brickDimensions[1]) System.out.println ("The aperture matches");
+        else System.out.println ("The aperture doesn't match");
```
Comment (arturlyup) [inline module_1/src/com/epam/BranchingTaskD.java:24]: Refactor if-else into if () { ...} else { ... }

Comment (arturlyup) [reply module_1/src/com/epam/BranchingTaskD.java:24]: BTW good solution!

File: module_1/src/com/epam/Mod1Task1.java [hunk 4]
```diff
@@ -1,16 +0,0 @@
...
-        Scanner console = new Scanner(System.in).useLocale(Locale.US);
-        System.out.print ("Enter the value a: ");
-        double a = console.nextDouble();
-        System.out.print ("Enter the value b: ");
-        double b = console.nextDouble();
-        System.out.print ("Enter the value c: ");
-        double c = console.nextDouble();
-        System.out.println ("Expression value = " + (((a - 3) * b / 2) + c));
-    }
-}
```
Comment (arturlyup) [inline module_1/src/com/epam/Mod1Task1.java:16]: return all these files and move to new package

File: module_1/src/com/epam/BranchingTaskD.java (hunk 3 above)
Comment (pmiakish) [reply module_1/src/com/epam/BranchingTaskD.java:24]: Thanks)

**Pull request merged**